pkgs/by-name/og/oggvideotools/package.nix

```diff
@@ -27,13 +27,29 @@ stdenv.mkDerivation rec {
       # svn patch, rely on prefix added by fetchpatch:
       extraPrefix = "";
     })
+
+    # Fixes compilation on Clang:
+    # src/effect/pictureBlend.cpp:28:11: error: no matching conversion for functional-style cast from 'const char[39]' to 'OggException'
+    #
+    # For some reason this is required on Clang but not GCC, most other files have string.h included via other headers anyways.
+    ./fix-clang.patch
```

Comment:
Why not upstream the patch?

Reply:
The last commit was 8 years ago. It looks like the main activity the project gets is @trofi [reporting compile failures](https://sourceforge.net/p/oggvideotools/bugs/). (Though other distros do package this, so it could potentially reduce some work to post the patch there if they’re checking. Most of them don’t care about Clang, though.)